### ISSUE TITLE
Pathing derps

### ISSUE THREAD

**Issue by SDMoonkeeper** [P3pp3rF1y/AncientWarfare2#420]
When my workers try to find a path to the town hall they derp around fences like they are trying to walk through them, stop moving, then just accept their fate and wait for salvation or death. It seams most likely with natura or agricraft fences but every once in a while it happens with vanilla fences. Bothersome when I decorate bottoms of houses or porches with fences....

**Comment by GotoLink** [comment]
Path finding mostly rely on "vanilla" code. Wouldn't be surprised if it did that for every minecraft mob.
Will try to find the bug.

**Comment by cosmicdan** [comment]
Yes I think it's related to vanilla code, just like how animals in a fenced farm stand at the fences and try to jump over it.

For some reason, vanilla path finding seems to consider blocks of 1.5 height (or whatever fences are) as valid "jump-up-able" blocks. I've tried searching the pathfinding code for something that might shed light, but it's way beyond me - namely because of the obfuscation/SRG stuff. Besides, if it were that simple I figure it would have been fixed by now by either Mojang or a veteran modder.
